webserver/src/state/state.cpp

```diff
@@ -7,6 +7,7 @@ namespace state {
   Profile activeProfile;
   ProfileId activeProfileId;
   OperationMode operationMode;
+  SystemState systemState;
```

Comment:
Hmm looks a bit weird now because OperationMode is contained inside the SystemState 🤔

Reply:
i see your point